package.json

```diff
@@ -34,59 +34,59 @@
     "@ianvs/prettier-plugin-sort-imports": "^4.7.0",
     "@storybook/react-vite": "^9.1.3",
     "@storybook/addon-vitest": "^9.1.3",
-    "@testing-library/jest-dom": "^6.8.0",
+    "@testing-library/jest-dom": "^6.9.1",
     "@testing-library/react": "^16.3.0",
     "@testing-library/user-event": "^14.6.1",
     "@types/history": "^4.7.11",
     "@types/jest": "^30.0.0",
-    "@types/node": "^22.17.1",
-    "@types/react": "^19.1.12",
-    "@types/react-dom": "^19.1.9",
+    "@types/node": "^22.18.13",
+    "@types/react": "^19.2.2",
+    "@types/react-dom": "^19.2.2",
     "@types/react-highlight": "^0.12.8",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.1.1",
     "@types/react-transition-group": "^4.4.12",
     "@types/wicg-file-system-access": "^2023.10.6",
-    "cspell": "^9.2.1",
+    "cspell": "^9.2.2",
     "jest": "^30.2.0",
     "jest-canvas-mock": "^2.5.2",
     "jsdom-testing-mocks": "^1.16.0",
-    "msw": "2.11.3",
-    "msw-storybook-addon": "^2.0.5",
+    "msw": "2.11.6",
+    "msw-storybook-addon": "^2.0.6",
     "node-gyp": "^11.4.2",
-    "playwright": "^1.55.1",
+    "playwright": "^1.56.1",
     "prettier": "^3.6.2",
     "react-select-event": "^5.5.1",
     "storybook": "^9.1.3",
     "svgo": "^4.0.0",
-    "typescript": "^5.9.2",
-    "vite": "^7.1.11"
+    "typescript": "^5.9.3",
+    "vite": "^7.1.12"
   },
   "dependencies": {
-    "@codemirror/autocomplete": "^6.19.0",
+    "@codemirror/autocomplete": "^6.19.1",
     "@codemirror/lang-sql": "^6.10.0",
-    "@codemirror/view": "^6.38.4",
+    "@codemirror/view": "^6.38.6",
     "@floating-ui/react": "^0.27.16",
     "@grpc/grpc-js": "1.14.0",
-    "@lezer/highlight": "^1.2.1",
+    "@lezer/highlight": "^1.2.3",
     "@nivo/bar": "^0.99.0",
     "@protobuf-ts/runtime": "^2.11.1",
     "@protobuf-ts/runtime-rpc": "^2.11.1",
-    "@tanstack/react-query": "^5.90.2",
+    "@tanstack/react-query": "^5.90.5",
     "@uiw/codemirror-themes": "^4.25.2",
     "@uiw/react-codemirror": "^4.25.2",
     "d3-scale": "^4.0.2",
     "d3-time-format": "^4.1.0",
     "date-fns": "^4.1.0",
     "highlight.js": "^11.11.1",
     "history": "^4.9.0",
-    "immer": "^10.1.3",
+    "immer": "^10.2.0",
     "prop-types": "^15.8.1",
-    "react": "^19.1.1",
-    "react-day-picker": "^9.11.0",
-    "react-dom": "^19.1.1",
+    "react": "^19.2.0",
```

Comment:
We've got to find a way to get v19.2.0 into v18. `useEffectEvent` is going to make certain rare situations so much simpler and let us remove [`useStateRef`](https://github.com/gravitational/teleport/blob/ceb43f935177a487a7bc66ec9abed55b97eeda5f/web/packages/shared/hooks/useStateRef.ts).